include/crocoddyl/core/activations/quadratic-flat.hpp

```diff
@@ -0,0 +1,145 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2020, LAAS-CNRS, University of Edinburgh
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_FLAT_HPP_
+#define CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_FLAT_HPP_
+
+#include <iostream>
```

Comment:
We shouldn't need to include iostream and stdexcept (utils/exceptions) is already included.